package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "veritable-ui",
-  "version": "0.11.7",
+  "version": "0.11.8",
```

Comment:
I feel this deserves a minor version bump

Reply:
agreed, will update in the next pushes